src/test/ui/symbol-names/basic.legacy.stderr

```diff
@@ -1,10 +1,10 @@
-error: symbol-name(_ZN5basic4main17hcbad207c0eeb0b3bE)
+error: symbol-name(_ZN5basic4main17he9f658e438f1cac0E)
```

Comment:
Yea, this is what I was concerned about, that seems dangerous. A type aliased thing should have the same symbol as just mentioning the thing directly.